src/templates/common/elements/profile_image.html

```diff
@@ -0,0 +1,6 @@
+{% load static %}
+<img
+    class="{{ classes }}"
+    src="{% if user.profile_image %}{{ user.profile_image.url }}{% elif request.journal.default_profile_image %}{{ request.journal.default_profile_image.url }}{% else %}{% static "common/img/icons/users.png" %}{% endif %}"
+    alt="{% if user.profile_image %}Headshot photograph of {{ user.full_name }}{% else %}A generic headshot photograph of a person{% endif %}"
```

Comment:
Trying to think of ways this might not suit all use cases--what if someone uploads a generic image rather than a headshot? Could we just say "Profile picture for ..."? (And of course we will have to revisit this when we have the capability for dynamic user-entered alt text.)